### PR TITLE
Updated with login-signup page

### PR DESCRIPTION
**Pull request by JaanviNL**
Screenshot (21)
Screenshot (23)

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -24,7 +24,8 @@
     height: 2rem;
     border-radius: 0.1rem;
     padding: 0.3rem;
-    border: 2px solid #71717a;
+    background-color: #f1f5f9 ;
```
Comment (Creator-singh) [inline index.css:27]: use variable here

File: navigation pages/login/login.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    margin-top: 4rem;
+    margin-left: 32rem;
+    padding: 1rem;
+    width: 22rem;
+    height: 25rem;
+    background-color: var(--background-form);
+    border: 0.1rem solid var(--primary-btn);
+    border-radius: 0.5rem;
+
+
```
Comment (Creator-singh) [inline navigation pages/login/login.css:26]: remove empty spaces there are multiple places where they are present

**Review by Creator-singh — APPROVED** [review]
else looks good to me

**Review by jeetbhalani15 — APPROVED** [review]
looks ok

**Pull request merged**